data/data-pipeline/data_pipeline/score/score_a.py

```diff
@@ -0,0 +1,13 @@
+from data_pipeline.score.score import *
```

Comment:
This is what's causing the lint to fail. I will update all these to name the classes/modules we need.